### PR TITLE
[terra-embedded-content-consumer] Update documentation to include accessibility guidance

### PR DESCRIPTION
**Pull request by kolkheang**
### Summary
Add a section around accessibility for the Embedded Content Consumer

**What was changed:**
Documentation update to include accessibility guidance

**Why it was changed:**

Provide guidance on the implementation to support accessibility guideline

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [x] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-8572

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-framework-docs/src/terra-dev-site/doc/embedded-content-consumer/EmbeddedContentConsumer.1.doc.mdx [hunk 1]
```diff
@@ -56,6 +57,23 @@ Consumer.init();
...
+
+#### Accessibility Guidance: Title
+
+For the purposes of accessibility compliance, it is **critical** to screen reader users that the title of the frame is set to a meaningful title for the content inside the iframe.
+
+#### Accessibility Guidance: Scrolling
+
+When using this Embedded Content Consumer component to embed another page, the default behavior is set to auto resize the content such that the content will display fully inside of the iframe. In scenario where custom configuration of XFC is made to set a `fixedWidth` or `fixedHeight` of the content, the content may not be fully visible in the viewport. For keyboard only users, it is critical that users can navigate to the iframe, and scroll to view the content fully.
+
+When specifying a `fixedWidth` or `fixedHeight` [attributes for XFC](https://github.com/cerner/xfc#iframe-resizing-config), it's also necessary to set `scrolling` option to `true` so that the content can be scrolled when not fully displayed. This is required for accessibility for keyboard only users.
```
Comment (RayGunY) [inline packages/terra-framework-docs/src/terra-dev-site/doc/embedded-content-consumer/EmbeddedContentConsumer.1.doc.mdx:72]: Nit, slight rephrase needed here, something like:

1. "When specifying `fixedWidth` or `fixedHeight` attributes for XFC"
2. "When specifying the `fixedWidth` or `fixedHeight` attributes for XFC"
3. "When specifying a `fixedWidth` or `fixedHeight` attribute for XFC"

**Comment by kolkheang** [comment]
@RayGunY  - I updated with more info. I also added a few things as previously suggested by @scottwilmarth .

**Review by eawww — APPROVED** [review]

**Review by scottwilmarth — APPROVED** [review]
Reviewed documentation updates and am approving.

**Review by RayGunY — APPROVED** [review]

**Comment by kolkheang** [comment]
@sdadn - can you approve and merge? It's only has a documentation update.

**Review by sdadn — APPROVED** [review]

**Pull request merged**